lib/tasks/sloc.rake

```diff
@@ -0,0 +1,41 @@
+# expose the commands used to calculate the Lines of Code. We are using Rake for
+# this to allow bamboo or other such systems to call one common action across
+# applications and get the similar responses
+# 
+# Usage:
+# rake sloc:cloc - run the cloc command report
+# rake sloc:sloc - run the sloc
+# rake sloc:report - expose to scripts which method should be run for MMT
+# 
+# @author thomas.a.cherry
+
+namespace :sloc do
+  desc 'use the cloc command to analyze the code, warn if not installed'
+  task :cloc do
+    cloc=`which cloc`
+    if cloc.empty?
+      puts 'run brew cloc before using this feature'
+    else
+   	  shell_cmd = 'cloc ' \
+        '--exclude-lang=Markdown,Dockerfile,"Bourne Shell" ' \
```

Comment:
I think comments explaining why these things are being excluded should be added. I think we have a grasp of why these are being done now, future us might not remember.

Reply:
will do